### ISSUE TITLE
Resize from left-top not possible

### ISSUE THREAD

**Issue by codinkai** [SabariVig/svelte-resize#1]
Hi,

I like the simplicity and the size of it. I have just tested your library and I do have an issue, when resizing from the top-left. Nothing happens. Maybe it is related to the abolute-positioning.

```
<script>
  import { resize } from 'svelte-resize';
</script>

<div style="background-color:aqua; position: absolute; top: 300px; left: 200px;" use:resize>
  Hallo Welt
</div>
```
Am I doing anything wrong? I want to resize a div as pictured in the README.

**Comment by SabariVig** [comment]
Hey, 
There was a undefined bug  which was causing the issue. I've pushed the fix to npm please use v1.0.3

**Comment by codinkai** [comment]
This is so cool. Thanks for fixing this issue so quickly :)
